setup.py

```diff
@@ -16,7 +16,7 @@
         "fallback_version": "0.0.0",
         "local_scheme": "no-local-version",
     },
-    setup_requires=["setuptools_scm"],
+    setup_requires=["setuptools_scm<8.0.1"],
```

Comment:
Temporary workaround - see https://github.com/pypa/setuptools_scm/issues/912

Reply:
bugfix incoming in https://github.com/pypa/setuptools_scm/pull/915